packages/contents/src/drivefs.ts

```diff
@@ -294,7 +322,14 @@ export class DriveFSEmscriptenNodeOps implements IEmscriptenNodeOps {
           const size = value;
           const path = this.fs.realPath(node);
           if (this.fs.FS.isFile(node.mode) && size >= 0) {
-            const file = this.fs.API.get(path);
+            let file;
+            try {
+              file = this.fs.API.get(path);
+            } catch (e) {
+              // TODO: Should do anything here? Should we create the file?
```

Comment:
If you guys don't mind, I'd like that we track this in an issue later. I'm not sure what we should do when `setattr` is called on a removed file. I'd be tempted to create the file but after briefly trying it seems to have some unwanted behaviors.